src/Grpc.AspNetCore.Server.HttpApi/Grpc.AspNetCore.Server.HttpApi.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Perhaps it would be good to add a README.md into this directory that clearly states that this is an experimental feature? Otherwise people might be confused that this functionality is the same level of stability as the other packages.